README.md

```diff
@@ -61,6 +67,77 @@ yarn start
 
 Finally, we can navigate a browser window to `localhost:8080` to view generated data.
 
+### Running with Docker
+
+Optionally, you can build and run sourcecred in a container to avoid installing dependencies
+on your host. First, build the container:
+
+```bash
+$ docker build -t sourcecred .
+```
+
+If you don't want to build, the container is also provided at [vanessa/sourcecred](https://hub.docker.com/r/vanessa/sourcecred)
+
+```bash
+$ docker pull vanessa/sourcecred
+$ docker tag vanessa/sourcecred sourcecred
+```
+
+You will still need to export a GitHub token, and then provide it to the container
+when you run it. Notice that we are also binding port 8080 so we can
+view the web interface that will be opened up.  The only argument needed is the 
+GitHub repository to generate the sourcecred for:
+
+```bash
+REPOSITORY=sfosc/sfosc
+$ docker run -d --name sourcecred --rm --env SOURCECRED_GITHUB_TOKEN=$(cat .token) -p 8080:8080 sourcecred $REPOSITORY
```

Comment:
Using `--env SOURCECRED_GITHUB_TOKEN="$(cat .token)"` s unsafe: it puts
the contents of the token in the command line, and the command lines of
processes are visible to _all users_ on the system.

The standard mechanism to pass secrets securely across process
boundaries is the environment, because the environment of a process is
not visible to other users. Docker supports this: passing `--env FOO`
instead of `--env FOO=bar` will pass the environment’s value of `FOO`
down to the container:

```shell
SOURCECRED_GITHUB_TOKEN="$(cat .token)" \
    docker run --env SOURCECRED_GITHUB_TOKEN …
```

Reply:
Good thing I'm the only user on my system :) But happy to make this change.

Reply:
Done.

Reply:
I’m also the only user on my system, except that I’m not:

```
$ ps -e -o user= | sort -u
avahi
colord
geoclue
messagebus
nobody
ntp
root
rtkit
syslog
uuidd
wchargin
```

For instance, `geoclue` is a service that determines my location and
provides it to applications like `redshift`. To do so, it may make
network requests against a GeoIP service. It shouldn’t be able to read
my token file.

(And of course lots of people use servers with multiple actual human
users, and we can’t just leak their credentials.)

Thanks for fixing. :-)